hooks/build

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements. See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership. The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# -----------------------------------------------------------------------------
+
+# we are on ubuntu xenial (16.4) java-11 is hard to install so we tage java 8
+apt-get update -y
+apt-get install default-jdk maven -y
+java -version
+mvn -version
+mvn clean package -P container
```

Comment:
Maybe this would be a good place to validate that:
- we don't build a snapshot version
- the base image name is the one we expect from the pom ( compare with `${IMAGE_NAME}` )